lib/logidze.rb

```diff
@@ -13,6 +13,8 @@ module Logidze
 
   require 'logidze/engine' if defined?(Rails)
 
+  mattr_accessor :preserve_future
```

Comment:
Let's avoid unnecessary ActiveSupport tricks. We have plans to make Logidze Rails-less (and ActiveSupport-less) in the future.